plenum/test/node_catchup/test_node_catchup_after_lost_connection.py

```diff
@@ -21,8 +23,10 @@ def testNodeCatchupAfterLostConnection(newNodeCaughtUp, txnPoolNodeSet,
     :return:
     """
     looper, newNode, client, wallet, _, _ = nodeSetWithNodeAddedAfterSomeTxns
-    logger.debug("Stopping node {} with pool ledger size {}".
-                 format(newNode, newNode.poolManager.txnSeqNo))
+    logger.debug("Disconnecting node {}, ledger size {}".
+                 format(newNode, newNode.domainLedger.size))
+    disconnect_node_and_ensure_disconnected(looper, txnPoolNodeSet, newNode,
```

Comment:
We have another test `test_node_catchup_after_disconnect.py` to check catch-up after a node is disconnected.
This test doesn't perform disconnect intentionally. It just removes A node from the Loop, so that a Node receives messages, but don't process them.